contrib/epee/src/mlog.cpp

```diff
@@ -28,6 +28,13 @@
 #ifndef _MLOG_H_
 #define _MLOG_H_
 
+#ifdef _WIN32
+#ifndef ENABLE_VIRTUAL_TERMINAL_PROCESSING
+#define ENABLE_VIRTUAL_TERMINAL_PROCESSING  0x0004
+#endif
+#include <windows.h>
+#endif
```

Comment:
Did you mean to test for the define before including ?

Reply:
ENABLE_VIRTUAL_TERMINAL_PROCESSING is not defined anywhere in MSYS2 MinGW.

Reply:
Ah, it is defined in newer MSYS2 MinGW. It was not at the time the commit was made.

Reply:
What is it supposed to be defined by then ? The user in the makefile ?